sys/mips/malta/malta_machdep.c

```diff
@@ -277,6 +277,44 @@ malta_cpu_freq(void)
 	return (platform_counter_freq);
 }
 
+
+/* Parse cmd line args as env - copied from beri_machdep. */
+static void
+_parse_bootargs(char *cmdline)
```

Comment:
Yet another copy of this... Any chance I can talk you into refactoring them all ?